### ISSUE TITLE
Pdigin ask me in loop the captcha, even correct

### ISSUE THREAD

**Issue by GoogleCodeExporter** [EionRobb/pidgin-opensteamworks#100]
```
What steps will reproduce the problem?
1.After configured Pidgin for the Steam chat with Steam ID and Steam Guard 
code, he ask for the Captcha. I gtive him the correct code, but after 30 
second, ask me again a new captcha
2.
3.

What is the expected output? What do you see instead?


What version of the product are you using? On what operating system?
Pidgin 2.10.9 on Linux Mint 17.2 x86, the the latest plugin version

Please provide any additional information below.

```

Original issue reported on code.google.com by `saggiora...@gmail.com` on 29 Jul 2015 at 8:14

**Comment by selurvedu** [comment]
I've got this problem as well. The plugin just keeps asking for captcha over and over.

**Comment by EionRobb** [comment]
@selurvedu can you check that the password is correct and that you're on v1.6.1 - if that fails can you attach the debug log from Help->Debug Window  as you try to connect to the account

**Comment by selurvedu** [comment]
@EionRobb Yep, you're right, thanks. I was entering the wrong password, but that was totally **not** obvious, and it should be.

Also, there's another issue: After I logged in, 32 new contacts appeared in my contact list. 17 of them were my steam friends, but another 15 are, well, let me describe.

- They are all offline.
- They don't have any nickname, only a set of 18 digits instead.
- Each number begins with "1035827914", so the contact's name looks like 1035827914xxxxxxxx.
- Right-clicking a contact and selecting "View online profile" opens a Steam Community page with error message telling me "The specified profile could not be found."

Searching for any of that IDs explains what they are – these are IDs of group chats. Can I somehow hide them without leaving my groups?

**Comment by EionRobb** [comment]
Yeah, unfortunately the steam servers don't say "wrong password" they just say "captcha required" and we don't know how to tell when it's actually required because the ip address has been greylisted or if it's because the password is wrong. 

The group ids are also a side effect of what the server sends us. They also show up as buddy friend requests when you get invited to a group! They should show as offline through? I should be able to add some code to detect whether it's a group though and hide it completely, maybe.

**Comment by Kiwii** [comment]
since pidgin supports group chats for jabber and irc, would it be possible to have it support steam group chats or would that be too much work or not possible due to a technical limitation of the protocol?

**Comment by EionRobb** [comment]
@Kiwii no, the Steam mobile/web protocol that we use doesn't support group chats yet unfortunately

**Comment by Kiwii** [comment]
okay, thanks. weird that they the group ids then but with all the weird stuff valve is doing in the mobile clients it doesn't really surprise me.
